components/Publication/PublicationForm.js

```diff
@@ -23,141 +16,14 @@ import {
   Field,
   Checkbox,
   A,
-  colors,
-  VariableContext
+  HR,
+  useColorContext
 } from '@project-r/styleguide'
 
 import MaskedInput from 'react-maskedinput'
-
-import { getRepoHistory, COMMIT_LIMIT } from '../../pages/repo/tree'
-import { getRepoWithPublications } from './Current'
-
-import { renderMdast } from 'mdast-react-render'
-
-import { getSchema } from '../../components/Templates'
-import RepoArchivedBanner from '../../components/Repo/ArchivedBanner'
-
+import { getSchema } from '../Templates'
 import useValidation from './useValidation'
 
-const publishMutation = gql`
-  mutation publish(
-    $repoId: ID!
-    $commitId: ID!
-    $prepublication: Boolean!
-    $scheduledAt: DateTime
-    $updateMailchimp: Boolean!
-    $ignoreUnresolvedRepoIds: Boolean
-    $notifySubscribers: Boolean
-  ) {
-    publish(
-      repoId: $repoId
-      commitId: $commitId
-      prepublication: $prepublication
-      scheduledAt: $scheduledAt
-      updateMailchimp: $updateMailchimp
-      ignoreUnresolvedRepoIds: $ignoreUnresolvedRepoIds
-      notifySubscribers: $notifySubscribers
-    ) {
-      unresolvedRepoIds
-      publication {
-        name
-      }
-    }
-  }
-`
```

Comment:
Re `publishMutation`: I mean that the mutation was removed from this file and added to the index file.

Reply:
I think the confusing part is: 
Previous file was PublishForm.js, which I renamed to "index.js"
It contained two components: `Form` and `Preview`
I removed Form from the file and created PublicationForm.js
Does that help?

Reply:
I see. I think I'd still move the publish mutation into the publish form (feels more modular that way). But that's kind optional.

Reply:
Ah I get it, I think you're right. Will do taht